src/Home.tsx

```diff
@@ -11,31 +12,39 @@ import {
   MediaQuery,
   Burger,
   useMantineTheme,
+  Modal,
 } from '@mantine/core';
+import Login from './Login';
+import breadImg from './assets/bread.png';
+import { useDisclosure } from '@mantine/hooks';
+
 
 export default function Home() {
   const theme = useMantineTheme();
   const [opened, setOpened] = useState(false);
+  const [modalOpened, {open, close}] = useDisclosure(false);
   
   const sayHi = (): void => {
     alert("Hello World!")
   }
 
   return (
-    <div>
+    <div className='tw-'>
```

Comment:
probably don't need this here

Reply:
Makes sense, everything is in appshell anyways, was a needless wrapper. kk